vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -137,8 +137,10 @@
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\JSCRuntime.cpp">
       <ExcludedFromBuild>true</ExcludedFromBuild>
     </ClCompile>
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\jsi.cpp" />
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\JSIDynamic.cpp" />
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\jsi\jsi\jsi.cpp" />
```

Comment:
Be sure to add any new entries to ReactCommon.vcxproj.filters

Reply:
Done.. in the new PR. I have to revert this one as i can't push updates to this branch.